src/1.4.x/lib/bindings.sc

```diff
@@ -70,7 +71,9 @@ class MarathonStorage(args: List[String] = helpers.InternalHelpers.argsFromEnv)
     case zk: CuratorZk => zk
   }
   lazy val store = {
-    val s = storage.store
+    val s: ZkPersistenceStore = underlyingModule.persistenceStore match {
```

Comment:
In 1.4.x the type is a bit different. It should be
```
    val s: ZkPersistenceStore = underlyingModule.persistenceStore match {
      case Some(persistenceStore: ZkPersistenceStore) => persistenceStore
    }
```

Reply:
Great catch! I'm not sure why this compiled.

Reply:
I checked it with the Some(...) and it works.